src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -9,6 +9,37 @@ interface Route {
   total_fees: number;
 }
 
+export interface Invoice {
+  id: string;
+  memo: string;
+  type: "received";
+  settled: boolean;
+  settleDate: number;
+  totalAmount: string;
+  totalAmountFiat?: string;
+  preimage: string;
+  custom_records?: {
+    "696969": string;
+    "7629169": string;
+    "5482373484": string;
+  };
+  boostagram?: {
```

Comment:
imo the connector should not know about boostagrams. 

The connector should return low-level lightning data and then in an action or some "higher level" we parse boostagrams, check fiat amounts, and other things.

Reply:
Good point. I split this up into two types.